doc/source/parameters/cfd/post_processing.rst

```diff
@@ -104,7 +108,10 @@ This subsection controls the post-processing other than the forces and torque on
     
 
 * ``calculate average velocities``: controls if calculation of time-averaged velocities is enabled.
-    * ``initial time``: initial time used for the average velocities calculations.
+    * ``initial time for average velocity``: initial time used for the average velocities calculations.
+
+* ``calculate average temperature and average heat flux``: controls if calculation of time-averaged temperature and time-averaged heat flux is enabled.
+    * ``initial time for average temperature``: initial time used for the average temperature calculations.
```

Comment:
I like that you made another initial time. That is very general. Cool stuff I have to admit. I am not sure many codes allow for this...